src/main/java/com/iexec/sms/tee/session/palaemon/PalaemonSessionService.java

```diff
@@ -223,6 +227,13 @@ public String getSessionYml(PalaemonSessionRequest request) throws Exception {
                 .filter(e -> e.getKey().contains(IexecEnvUtils.IEXEC_INPUT_FILE_NAME_PREFIX))
                 .collect(Collectors.toMap(e -> e.getKey(), e -> e.getValue()));
         tokens.put(INPUT_FILE_NAMES, inputFileNames);
+
+        // Add application runtime secrets
+        final long secretIndex = 0;
+        applicationRuntimeSecretService.getSecret(taskDescription.getAppAddress(), secretIndex, true)
```

Comment:
Are we sure appAddress is not null, empty or 0x?

Reply:
It should not be null nor empty as it is directly retrieved at the same time as the other fields of `taskDescription`.
Regarding a potentail `0x` prefix, I don't know. I thought all app address should start with `0x`. Is this assumption wrong? If it is, we should sanitize it everywhere in app runtime secret management so we ensure we always - or never - have `0x`.

Reply:
Yes, let's rely on `Optional<ChainApp> getChainApp(App app)`